promscale/about-promscale.md

```diff
@@ -0,0 +1,228 @@
+Connector writes data to TimescaleDB.
+
+SQL queries are handled directly by TimescaleDB. PromQL queries can be directed
+to the Promscale Connector. Alternatively, you can direct PromQL queries to the
+Prometheus instance, which reads data from the Connector using the `remote_read`
+interface. The Connector, in turn, fetches data from TimescaleDB. 
+
+## Architecture
+Promscale includes two components:
+
+**Promscale Connector**: a stateless service that provides the ingest interfaces
+for observability data, processes that data and stores it in TimescaleDB. It
+also provides an interface to query the data with PromQL. The Promscale
+Connector automatically sets up the data structures in TimescaleDB to store the
+data and handles changes in those data structures required if required for
+upgrading to newer versions of Promscale.
+
+**TimescaleDB**: the Postgres-based database where all the observability data is
+stored. It offers a full SQL interface for querying the data as well as advanced
+capabilities like analytical functions, columnar compression and continuous
+aggregates. TimescaleDB offers a lot of  flexibility to also store business and
+other types of data that you can then use to correlate with observability data.
+
+The Promscale Connector ingests Prometheus metrics, metadata and OpenMetrics
+exemplars using the Prometheus `remote_write` interface. It also ingests
+OpenTelemetry traces using the OpenTelemetry protocol (OTLP). It can also ingest
+metrics and traces in other formats using the OpenTelemetry Collector to process
+and send them over the Prometheus `remote_write` interface and the OpenTelemetry
+protocol. For example, you can use the OpenTelemetry Collector to ingest Jaeger
+traces and StatsD metrics into Promscale.
+
+For Prometheus metrics, the Promscale Connector exposes Prometheus API endpoints
+for running PromQL queries and reading metadata. This allows you to connect
+tools that support the Prometheus API, such as Grafana, directly to Promscale
+for querying. It's also possible to send queries to Prometheus and have
+Prometheus read data from Promscale using the Promscale Connector on the
+`remote_read` interface.
+
+For OpenTelemetry traces, there is a Jaeger storage plugin that implements the
+interface for querying and retrieving traces. This allows you to visualize
+traces stored in Promscale in Jaeger as well as Grafana by configuring a Jaeger
+data source. In this case, Grafana queries Jaeger, which then queries Promscale.
+
+You can also query metrics and traces in Promscale using SQL which allows you to
+use many different visualization tools that integrate with PostgreSQL. For
+example, Grafana supports querying data in Promscale using SQL out of the box
+through the PostgreSQL data source.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/promscale-arch.png" alt="Promscale architecture diagram"/>
```

Comment:
This image feels like it comes quite late relative to the text.

Reply:
moved it up to L50